cunumeric/module.py

```diff
@@ -4646,12 +4646,12 @@ def cumprod(
         Input array.
 
     axis : int, optional
-        Axis along which the cumulative sum is computed. The default (None) is
-        to compute the cumsum over the flattened array.
+        Axis along which the cumulative product is computed. The default (None)
+        is to compute the cumprod over the flattened array.
 
     dtype : dtype, optional
         Type of the returned array and of the accumulator in which the elements
-        are summed. If dtype is not specified, it defaults to the dtype of a,
+        are returned. If dtype is not specified, it defaults to the dtype of a,
```

Comment:
I think this should be "multiplied"

Reply:
a3a8593fed3c1f0534e2d2dc251d75967b9790c3